libclamav_rust/src/css_image_extract.rs

```diff
@@ -22,6 +22,7 @@
 
 use std::{ffi::CStr, mem::ManuallyDrop, os::raw::c_char};
 
+use base64::{Engine as _, engine::general_purpose};
```

Comment:
This is a case where renaming the import might make sense, since it's not super-clear what a `general_purpose::STANDARD` does below (without the context of an import that occurs 200 lines above).  You could try something like this (as a rather verbose example):
```
use base64::{Engine as _, engine::general_purpose as base64_engine_standard};
```

Reply:
I totally agree.  I copypasted it from the base64 documentation and hated it. Thanks for the great idea!